config/test.exs

```diff
@@ -3,6 +3,19 @@ import Config
 # Import runtime configuration
 import_config "releases.exs"
 
+defmodule TestEnvironment do
+  @database_name_suffix "_test"
+
+  def enforce_test_database_name(key) do
+    url = Environment.get_local_url(key)
+
+    case String.contains?(url, @database_name_suffix) do
+      true -> url
+      _ -> raise "Expected database URL to ends with '#{@database_name_suffix}', got: #{url}"
+    end
```

Comment:
We should use `if` since it’s a simple _boolean_ check. 

```suggestion
    if String.contains?(url, @database_name_suffix) do
      url
    else
      raise "Expected database URL to ends with '#{@database_name_suffix}', got: #{url}"
    end
```